### PR TITLE
[circle-quantizer] Add support for MirrorPad

### PR DESCRIPTION
**Pull request by ai-moiseev**
This commit adds support for MirrorPad quantization.

ONE-DCO-1.0-Signed-off-by: Alexander Moiseev <a.moiseev@samsung.com>

### CODE REVIEW

**Review by binarman — APPROVED** [review]
LGTM

**Review by kvochko — APPROVED** [review]
LGTM

**Comment by binarman** [comment]
FYI @seanshpark @jinevening

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -925,6 +925,7 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
     case luci::CircleOpcode::LOCAL_RESPONSE_NORMALIZATION:
     case luci::CircleOpcode::MEAN:
     case luci::CircleOpcode::PAD:
+    case luci::CircleOpcode::MIRROR_PAD:
     case luci::CircleOpcode::REDUCE_ANY:
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:929]: This is NOT alphabetical order. Please be aware and post another PR to FIX THIS!

Comment (ai-moiseev) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:929]: Sorry, I've noticed this too late. Here's PR: #6870
